src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -73,79 +75,40 @@ export default class StripeV3PaymentStrategy implements PaymentStrategy {
         }
 
         if (isVaultedInstrument(paymentData)) {
-            try {
-                paymentPayload = {
-                    methodId,
-                    paymentData: {
-                        formattedPayload: {
-                            bigpay_token: {
-                                token: paymentData.instrumentId,
-                            },
-                            credit_card_number_confirmation: paymentData.ccNumber,
-                            verification_value: paymentData.ccCvv,
-                            confirm: false,
-                        },
-                        shouldSetAsDefaultInstrument,
-                    },
-                };
-
-                return await this._store.dispatch(this._paymentActionCreator.submitPayment(paymentPayload));
-            } catch (paymentError) {
-                const isThreeDSecureRequiredError = paymentError instanceof RequestError &&
-                    some(paymentError.body.errors, { code: 'three_d_secure_required' });
-
-                if (!isThreeDSecureRequiredError) {
-                    return Promise.reject(paymentError);
-                }
-
-                const clientSecret = paymentError.body.three_ds_result.token;
-
-                paymentResult = await this._confirmVaultedPayment(clientSecret);
-
-                if (!paymentResult.id) {
-                    throw new MissingDataError(MissingDataErrorType.MissingPaymentToken);
-                }
-
-                paymentPayload = {
-                    methodId,
-                    paymentData: {
-                        formattedPayload: {
-                            credit_card_token: {
-                                token: paymentResult.id,
-                            },
-                            confirm: true,
-                        },
-                    },
-                };
-            }
+            // tslint:disable-next-line: variable-name
```

Comment:
why is this required?

Reply:
Because linter complains about `credit_card_number_confirmation`
and `verification_value`. Do you want me to remove it?